packages/core/src/browser/style/tabs.css

```diff
@@ -261,22 +262,14 @@ body.theia-editor-highlightModifiedTabs
   padding-right: 4px;
 }
 
-.p-TabBar.theia-app-centers .p-TabBar-tab.p-mod-current > .theia-tab-icon-label > .p-TabBar-tabIcon {
-  margin-top: -2px;
-}
-
-.p-TabBar.theia-app-centers .p-TabBar-tab.p-mod-current > .p-TabBar-tabCloseIcon {
-  margin-top: 0px;
-}
```

Comment:
@msujew I'm not sure why the changes were necessary for the current tab but it caused the shifting of the `file-icon` and the `close` to occur:

![master-tab](https://user-images.githubusercontent.com/40359487/156241236-31ff757f-bc29-4fd5-9ec9-d5b8ea12571e.gif)

Reply:
Interesting, that only seems to happen on Chrome. I tested this (and build the original PR) on Firefox, where these lines of CSS fixed the shifting issue. But it seems like they introduced shifting in Chrome